modules/table/src/integrationTest/java/org/apache/ignite/distributed/ItInternalTableScanTest.java

```diff
@@ -343,7 +348,7 @@ public void testExceptionRowScanCursorHasNext() throws Exception {
 
             doAnswer(
                     invocationClose -> {
-                        cursorClosed.set(true);
+                        cursorClosedLatch.countDown();
```

Comment:
There is the same as above: only one waiting expression is enough.

Reply:
There is not an issue.

Reply:
Was discussed explicitly.